<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1456</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>381</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
position: absolute;
top: 50%;
left: 50%;
transform: translate(-50%, -50%);
font-size: 16px;
color: black;     
border: 3px solid rgb(240, 240, 249);   
background-color:#F5F5F5; 
border-radius: 5px;       
padding: 5px;   

</string>
   </property>
   <property name="title">
    <string>Registration</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>331</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="password_1">
       <property name="styleSheet">
        <string notr="true">border:none;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password">
       <property name="styleSheet">
        <string notr="true">border: 2px solid #3498db; /* Border thickness and color */
border-radius: 5px;       /* Optional: rounded corners */
padding: 5px;             /* Optional: padding inside the QLineEdit */
</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f57224; 
color: black;            
border: none;              
border-radius: 5px;       
padding: 5px;   </string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f57224; 
color: black;            
border: none;              
border-radius: 5px;       
padding: 5px;   </string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>331</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="username_1">
       <property name="styleSheet">
        <string notr="true">border:none;
font-size: 16px;




</string>
       </property>
       <property name="text">
        <string>User Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="username">
       <property name="styleSheet">
        <string notr="true">border: 2px solid #3498db; 
border-radius: 5px;      
padding: 5px;            
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>331</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="password_2">
       <property name="styleSheet">
        <string notr="true">border:none;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>Re.Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_3">
       <property name="styleSheet">
        <string notr="true">border: 2px solid #3498db; /* Border thickness and color */
border-radius: 5px;       /* Optional: rounded corners */
padding: 5px;             /* Optional: padding inside the QLineEdit */
</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
